release/dist/qnap/files/scripts/build-qpkg.sh

```diff
@@ -7,6 +7,14 @@ function cleanup() {
     rm -rf /Tailscale/$ARCH
     rm -f /Tailscale/sed*
     rm -f /Tailscale/qpkg.cfg
+
+    # If this build was signed, a .qpkg.codesigning file will be created as an
+    # artifact of the build
+    # (see https://github.com/qnap-dev/qdk2/blob/master/QDK_2.x/bin/qbuild#L992).
```

Comment:
permalink, in case that file changes and lines move around
```suggestion
    # (see https://github.com/qnap-dev/qdk2/blob/93ac75c76941b90ee668557f7ce01e4b23881054/QDK_2.x/bin/qbuild#L992).
```